### ISSUE TITLE
Add logging when a page is visited on web side.

### ISSUE THREAD

**Issue by viperfx** [redwoodjs/graphql#1527]
As the application grows, and as usage of multiple cells and queries within a page grows, it's easy to lose track of how many GraphQL queries are being executed on a page. 

I would like to see a log of a route change to the web side.

Note the addition of **web | GET /page 200 95.673 ms - 200**

```
web | GET /page 200 95.673 ms - 200
api | POST /graphql 200 95.673 ms - 200
api | POST /graphql 200 57.315 ms - 200
api | POST /graphql 200 457.261 ms - 1672
web | GET /page2 200 95.673 ms - 200
api | POST /graphql 200 95.673 ms - 200
```

**Comment by dthyresson** [comment]
> it's easy to lose track of how many GraphQL queries are being executed on a page.


> I would like to see a log of a route change to the web side.

These seem like two issues with different possible solutions.

Logging to the development dev server output in dev assumes that the request are sequential and no other route or page will load and initiate graphql calls while the first "get" has finished. That is, you won't necessarily be able to guarantee that counting the /grahpql posts means the prior web route initiated it.

If you want to log page loads, you could add a `useEffect` to the main layout that uses `useLocation` to fetch the location and log the pathname and/or search query string:

https://github.com/redwoodjs/redwood/blob/6cf2a9a38c73dbf82d7ec1c75479eec3f7234143/packages/router/src/location.js#L17

But that output would appear in the browser console, not in the dev server output.

If you wanted to count graphql calls, might not the logging be better done on the API side and output the event request that invoked the resolver?

**Comment by viperfx** [comment]
> If you wanted to count graphql calls, might not the logging be better done on the API side and output the event request that invoked the resolver?

Yes, that's prob a better idea. I suppose this is on the larger topic of how do we keep aware of how expensive the graphql calls are? I think not just counting the calls, but also timing wise and more SQL optimization. Maybe there is another tool with graphql to do this? 

This could just be premature optimization on my part.
